install/kubernetes/helm/istio-remote/charts/security/templates/serviceaccount.yaml

```diff
@@ -2,7 +2,9 @@ apiVersion: v1
 kind: ServiceAccount
 {{- if .Values.global.imagePullSecrets }}
 imagePullSecrets:
-{{ toYaml .Values.global.imagePullSecrets | indent 2 }}
+{{- range .Values.global.imagePullSecrets }}
```

Comment:
Shouldn't this be only on Pods ? What does it mean for a ServiceAccount ?

Reply:
@costinm putting `imagePullSecrets` to `serviceAccount` is more flexible, as users will not need to update every pod by adding such configuration.